internal/capability/operator_install.go

```diff
@@ -49,7 +49,7 @@ func OperatorInstall(s operator.SubscriptionData, c operator.Client) error {
 
 	logger.Debugw("installing package", "package", s.Package, "channel", s.Channel, "installmode", s.InstallModeType)
 
-	namespace := strings.Join([]string{"opcap", s.Package, s.Channel, strings.ToLower(string(s.InstallModeType))}, "-")
+	namespace := strings.Join([]string{"opcap", strings.ReplaceAll(s.Package, ".", "-")}, "-")
```

Comment:
Hi @yashoza19 , I notice some package names weirdly coming in upper case letters. According to [RFC 4343](https://datatracker.ietf.org/doc/html/rfc4343#section-3) it shouldn't be a problem if in a DNS request but Kubernetes objects should be lowercase named according to this https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names. What I see here is that you dropped the ToLower function. Can we have it back just as an insurance term?

Reply:
Ok. @yashoza19 confirmed all package names are lower case. We can proceed.
/approve